std/conv.d

```diff
@@ -5611,26 +5611,38 @@ auto toChars(ubyte radix = 10, Char = char, LetterCase letterCase = LetterCase.l
          * ulong.max is 1844_6744_0737_0955_1615
          *  long.max is  922_3372_0368_5477_5807
          */
-        struct Result
+        static struct Result
         {
-            this(UT value)
+            void initialize(UT value)
             {
                 bool neg = false;
-                if (value < 0)
+                if (value < 10)
                 {
+                    if (value >= 0)
```

Comment:
Hmm. Does it make sense to also optimize single digit negative numbers? Or the added complexity would outweigh the benefits?